RELEASE.md

```diff
@@ -41,58 +41,70 @@ with `make upgrade-npm-deps`
 
 ### 1. Prepare your release
 
-You should start to create a branch that follows the pattern `release/v<X.Y>`. Release candidates and patch releases
-for any given major or minor release happen in the same `release/v<major>.<minor>` branch. Do not
-create `release/<version>` for patch or release candidate releases.
+#### Create a release branch
 
-- Do this in a proper PR pointing to the release branch as this gives others the opportunity to chime in on the release
-  in general and on the addition to the changelog in particular.
+Create a branch that follows the naming pattern `release/v<X.Y>` and includes the changes you intend to release (usually the latest from `main`). Push it to github. You will use this branch as the base in the next step.
 
-- Update the file `VERSION` with the new version to be created.
-
-- Update the file `CHANGELOG.md` :
-
-```bash
-make generate-changelog
-```
-
-Note: it will generate a first changelog version based on the git history. You should review what has been generated.
-Error can happen ;)
-
-- Entries in the `CHANGELOG.md` are meant to be in this order:
-
-* `[FEATURE]`
-* `[ENHANCEMENT]`
-* `[BUGFIX]`
-* `[BREAKINGCHANGE]`
-
-As we have many libraries we publish, it's better if you also put a clear indication about what library is affected by
-these changes.
+> ⚠️ Release candidates and patch releases for any given major or minor release happen in the same `release/v<major>.<minor>` branch. Do not create `release/<version>` for patch or release candidate releases.
 
-- Update the different `package.json` with the corresponding version:
+#### Create a PR with the changes
 
-```bash
-make bump-version
-```
+- Create a branch using the release branch `release/v<X.Y>` you just created in the step above (e.g. `yourname/release-v<major>.<minor>.<patch>`).
```

Comment:
Create a branch from (instead of using)?

Reply:
I might say: "Create a branch from... [rest of sentence]. You might name it something like: `yourname/release-v<major>.<minor>.<patch>`"

The reason is I say this is that as I read this sentence, I got confused and thought the release branch was called `yourname/...`